force-app/main/default/classes/ContactEditFormController.cls

```diff
@@ -96,13 +102,16 @@ public with sharing class ContactEditFormController {
 		if (credentialToInsert == null) {
 			throw new AuraHandledException('Wrong credential');
 		}
-
 		SecureDml insertObject = new SecureDml();
 		try{
 			insertObject.dmlInsert(credentialToInsert);
 		}catch (Exception e){
 			throw new AuraHandledException(JSON.serialize(e.getTypeName() + ': ' + e.getMessage() + ' - ' + e.getStackTraceString()));
 		}
+		/*Database.SaveResult saveResult = Database.insert(credentialToInsert, false);
```

Comment:
no more need for this part of the code? better to just delete ^^

Reply:
Removed comments